layouts/partials/favicons.html

```diff
@@ -0,0 +1,9 @@
+<link rel="shortcut icon" href="{{ "favicons/favicon.ico" | relURL }}" >
+<link rel="apple-touch-icon" href="{{ "favicons/apple-touch-icon.png" | relURL }}" sizes="180x180">
+<link rel="icon" type="image/png" href="{{ "favicons/favicon-16x16.png" | relURL }}" sizes="16x16">
+<link rel="icon" type="image/png" href="{{ "favicons/favicon-32x32.png" | relURL }}" sizes="32x32">
+<link rel="manifest" href="{{ "favicons/site.webmanifest" | relURL }}">
+<link rel="mask-icon" href="{{ "favicons/safari-pinned-tab.svg" | relURL }}" color="#333333">
+<meta name="msapplication-config" content="{{ "favicons/browserconfig.xml" | relURL }}" />
+<meta name="msapplication-TileColor" content="#333333">
```

Comment:
These seem like they were specific to Internet Explorer. Since IE11 is now end-of-life, do we need these?

Reply:
I know, but there are people who use it. I think it will only be a plus if it is